index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="container1">
+        <a href="#logo" class="logo"><img src="../img/moyo_1.png" alt="img"></a>
+        <nav class="nav">
+          <ul class="nav_list">
+            <li class="is-active">
+              <a class="nav_link is-active" href="apple.asp">Apple</a>
+            </li>
+            <li class="nav_item"><a href="samsung.asp" class="nav_link">Samsung</a></li>
+            <li class="nav_item"><a href="smartphones.asp" class="nav_link">Smartphones</a></li>
+            <li class="nav_item">
+              <a href="laptops & computers.asp" class="nav_link laptops"
+            id="shows_activity" data-qa="hover">Laptops & Computers</a>
+            </li>
+            <li class="nav_item"><a href="gadgets.asp" class="nav_link">Gadgets</a></li>
+            <li class="nav_item"><a href="tablets.asp" class="nav_link">Tablets</a></li>
+            <li class="nav_item"><a href="photo.asp" class="nav_link">Photo</a></li>
+            <li class="nav_item"><a href="video.asp" class="nav_link">Video</a></li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="container2">
+        <a href="#logo" class="logo"><img src="../img/moyo_1.png" alt="img"></a>
+        <nav class="nav">
+          <ul class="nav_list">
+            <li class="is-active">
+              <a class="nav_link nav1 is-active" href="apple.asp">Apple</a>
+            </li>
+            <li class="nav_item"><a href="samsung.asp" class="nav_link">Samsung</a></li>
+            <li class="nav_item"><a href="smartphones.asp" class="nav_link">Smartphones</a></li>
+            <li class="nav_item ">
+              <a href="laptops & computers.asp" class=" nav_link laptops shows_activity"
+             data-qa="hover">Laptops & Computers</a>
+            </li>
+            <li class="nav_item"><a href="gadgets.asp" class="nav_link">Gadgets</a></li>
+            <li class="nav_item"><a href="tablets.asp" class="nav_link">Tablets</a></li>
+            <li class="nav_item"><a href="photo.asp" class="nav_link">Photo</a></li>
+            <li class="nav_item"><a href="video.asp" class="nav_link">Video</a></li>
+          </ul>
+        </nav>
+      </div>
```

Comment:
remove it, should be one responsive container

Reply:
I do it